NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -485,6 +487,53 @@ class NeuroID
             }
         }
 
+        /**
+         * ported from the iOS implementation
+         */
+        override fun startAppFlow(
+            siteID: String,
+            userID: String?,
```

Comment:
Have you merged in master because this shouldn't be new should it?

Reply:
my bad, I was viewing changes since my last review and this showed which is what confused me, when I looked holistically it was fine